opentracing-util/src/main/java/io/opentracing/util/ThreadLocalActiveSpanSource.java

```diff
@@ -36,7 +34,12 @@ public ThreadLocalActiveSpan activeSpan() {
 
     @Override
     public ActiveSpan makeActive(Span span) {
-        return new ThreadLocalActiveSpan(this, span, new AtomicInteger(1));
+        return new ThreadLocalActiveSpan(this, span, new AutoFinisher());
```

Comment:
I tend to agree with @bhs that defaulting no-arg `startActive()` to auto-finish/refcounting behavior is a better default for most users. I don't have any data to back that up since our use of Java is relatively limited. But according to this [search](https://github.com/search?l=Maven+POM&q=%22opentracing-api%22+%220.30%22+%22opentracing-contrib%22&type=Code&utf8=%E2%9C%93), a number of opentracing-contrib integration libraries have been upgraded to 0.30 using default auto-finish mechanism, so apparently it is both useful and convenient to use in instrumentation. @pavolloffay - any comments on that?